From david-b@pacbell.net  Thu Jul  2 15:46:21 2009
From: Kevin Hilman <khilman@deeprootsystems.com>
Date: Wed, 1 Jul 2009 03:33:46 -0700
Subject: USB: musb: davinci dm6446evm GPIO renumbering
To: Greg KH <greg@kroah.com>
Cc: linux-usb@vger.kernel.org, Kevin Hilman <khilman@deeprootsystems.com>
Message-ID: <200907010333.46803.david-b@pacbell.net>
Content-Disposition: inline


From: Kevin Hilman <khilman@deeprootsystems.com>

Numbering for GPIOs on the pcf857x chips on the dm644x EVM board
changed when DaVinci chips with more GPIOs were supported.  Update
the GPIO number used for nVBUS_DRV.

Longer term, we need a better abstraction of board-specific setup in
this code so we're not hard-coding board specific GPIOs into the
driver, but for now this at least gets it back to working with
mainline davinci core code.

Signed-off-by: Kevin Hilman <khilman@deeprootsystems.com>
Signed-off-by: Greg Kroah-Hartman <gregkh@suse.de>

---
 drivers/usb/musb/davinci.c |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

--- a/drivers/usb/musb/davinci.c
+++ b/drivers/usb/musb/davinci.c
@@ -42,7 +42,7 @@
 #include "musb_core.h"
 
 #ifdef CONFIG_MACH_DAVINCI_EVM
-#define GPIO_nVBUS_DRV		87
+#define GPIO_nVBUS_DRV		144
 #endif
 
 #include "davinci.h"
